CHANGES.txt

```diff
@@ -1,4 +1,5 @@
 Current
+New: Yaml parser: implement loadClasses flag (Dzmitry Sankouski)
```

Comment:
Just for traceability, it would be good if you could please help do the following:

1. Log a bug that explains the problem
2. Refer to that bug id here so that the changeset information is available in the `CHANGES.txt`
3. Add a reference to the issue in the `description` attribute of `@Test` test, so that we know that the test belongs to a specific defect.

Reply:
done